src/Streamly/System/Process.hs

```diff
@@ -33,7 +33,7 @@
 -- >>> import qualified Streamly.System.Process as Process
 -- >>> import qualified Streamly.Unicode.Stream as Unicode
 -- >>> import qualified Streamly.Internal.FileSystem.Dir as Dir
--- >>> import qualified Streamly.Data.Stream.Concurrent as Concur
+-- >>> import qualified Streamly.Data.Stream.Prelude as Concur
```

Comment:
Can import this as "Stream" and can remove "import qualified Streamly.Internal.Data.Stream as Stream".